### PR TITLE
[ADD] pos_print_receipt_backend

### PR DESCRIPTION
**Pull request by vvrossem**
[Task](https://gestion.coopiteasy.be/web#id=5142&view_type=form&model=project.task&menu_id=338&action=479)

A new module that allows the user to print POS receipts from the Odoo Backend.
The design of the receipt is inspired by the one used for `PosTicket` in `point_of_sale`.

Not all elements usually displayed in a POS receipt are available since `pos.order` fields differ from `Order` and `Orderline` JS objects.

Roadmap:
- [ ] implement _Tax details_
- [ ] implement _Total discount_
- [ ] generate `.po`

### POS Receipt:
receipt-pos

### PDF Receipt from Odoo Backend:
[receipt-backend.pdf](https://github.com/coopiteasy/cie-pos/files/5494132/receipt-backend.pdf)

### CODE REVIEW

**Comment by vvrossem** [comment]
closing in favor of [OCA's PR](https://github.com/OCA/web/pull/1723/)

**Comment by vvrossem** [comment]
oops, wrong PR :D

**Review by robinkeunen — CHANGES_REQUESTED** [review]
Faudrait passer un petit coup de pre-commit dessus, sinon, les rapports ont l'air bien, pas grand chose à dire :p

**Comment by vvrossem** [comment]
@robinkeunen `pre-commit run --all-files` effectué.
PS : j'ai mis à jour la description de la PR avec le rendu du ticket, spéciale dédicasse à @remytms

**Review by robinkeunen — APPROVED** [review]

**Comment by houssine78** [comment]
@vvrossem you're trolling @remytms with the list of products !

File: pos_print_receipt_backend/views/pos_order_view.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_pos_order_form" model="ir.ui.view">
+        <field name="model">pos.order</field>
+        <field name="inherit_id" ref="point_of_sale.view_pos_pos_form"/>
+        <field name="arch" type="xml">
+            <field name="state" position="before">
+                <button name="%(action_report_receipt)d" string="Receipt" attrs="{'invisible': [('state', '=', 'draft')]}" type="action" groups="point_of_sale.group_pos_user"/>
```
Comment (houssine78) [inline pos_print_receipt_backend/views/pos_order_view.xml:8]: a button and an email template to send it by email would be great ! it make it paperless and especially if the customer request the ticket by email and not in the shop.

**Review by houssine78 — APPROVED** [review]

**Comment by houssine78** [comment]
@vvrossem  a button and an email template to send it by email would be great ! it make it paperless and especially if the customer request the ticket by email and not in the shop.

**Comment by remytms** [comment]
@vvrossem Nice troll. XD

**Pull request merged**